.github/workflows/test-assistant-pr-scrub.yml

```diff
@@ -0,0 +1,34 @@
+name: Send PRs that require further testing to Slack
+on:
+  pull_request_target:
+    types: [labeled]
+
+permissions: {}
+
+jobs:
+  send-pr-to-slack-for-further-testing:
+    if: "${{ github.event.label.name == 'needs: analysis' }}"
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Get the PR status
+        id: get_pr_status
+        run: |
+          pr_status=${{ github.event.pull_request.merged == true && 'merged' || 'open' }}
+          echo "pr_status=$pr_status" >> $GITHUB_OUTPUT
+      - name: Calculate SLA timestamp
+        id: calculate_sla
+        run: |
+          sla_timestamp=$(date -d "${{ github.event.label.created_at }} + 5 hours" -u +%s)
+          echo "sla_timestamp=$sla_timestamp" >> $GITHUB_OUTPUT
+      - name: Notify Slack
+        uses: archive/github-actions-slack@d9dae40827adf93bddf939db6552d1e392259d7d
+        id: notify
+        with:
+            slack-bot-user-oauth-access-token: ${{ secrets.TEST_ASSISTANCE_BOT_TOKEN }}
+            slack-channel: ${{ secrets.WOO_PR_TESTING_ANALYSIS_SLACK_CHANNEL }}
+            slack-text: |
+                <!subteam^${{ secrets.WOO_PR_TESTING_ANALYSIS_TEAM_ID }}>, there is a new *${{ steps.get_pr_status.outputs.pr_status }}* PR to analyze.
+                <${{ github.event.pull_request.html_url }}|${{ github.event.pull_request.title }}> :thread:
+                *SLA:* <!date^${{ steps.calculate_sla.outputs.sla_timestamp }}^{date_short_pretty} at {time}|5 hours from now>
```

Comment:
Some of this text looks like it might be tokens for Slack. Does this `date_short_pretty` get localized? Otherwise the time provided by the Workflow (probably UTC?) might not be super useful for readers. Perhaps just having folks use the timestamp in Slack might be more reliable?

Reply:
Yes! The time is localized, we explicitly tested it. Thanks for highlighting this @ObliviousHarmony! It was a strong point to keep in mind while creating this workflow.

Reply:
Glad to hear!